build/common.sh

```diff
@@ -242,11 +242,15 @@ function kube::build::is_osx() {
   [[ "$(uname)" == "Darwin" ]]
 }
 
+function kube::build::is_gnu_sed() {
+  [[ $(sed --version 2>&1) == *GNU* ]]
```

Comment:
Have you tested this on Mac OS X?  BSD based `sed` there gives an error and exits with 1 with `sed --version`.

```
$ sed --version
sed: illegal option -- -
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
$ echo $?
1
```

I'm afraid this test will trigger errexit on the mac.